src/ServiceProviderLaravel5.php

```diff
@@ -24,9 +24,9 @@ public function boot()
      */
     public function register()
     {
-        $this->mergeConfigFrom(__DIR__.'/config/config.php', 'slack');
+        $this->mergeConfigFrom(__DIR__ . '/config/config.php', 'slack');
 
-        $this->app['maknz.slack'] = $this->app->share(function ($app) {
+        $this->app['maknz.slack'] = $this->app->singleton('maknz.slack', function ($app) {
```

Comment:
I don't think this will work . You don't need to reassign `$this->app['maknz.slack']` with the returned value from `app->singleton` since you already define the name for the singleton as a parameter to the function

Reply:
Could you please review my change and accept, have tested locally over tinker, Slack command works